tests/regression/roles/sonic_ospfv3/tasks/tasks_template.yaml

```
@@ -0,0 +1,24 @@
+---
+- name: "{{ item.name ~ ' , ' ~ item.description }}"
+  dellemc.enterprise_sonic.sonic_ospfv3:
+    config: "{{ item.input }}"
+    state: "{{ item.state }}"
+  register: action_task_output
+  failed_when: false
```

```suggestion
  ignore_errors: true
```

Replace failed_when with ignore_errors across the yaml file.